app.js

```diff
@@ -1,16 +1,99 @@
-/* Imports */
+// function getRandomItem(array) {
+//     const random = getRandomNumber(array.length);
+//     const item = array[random];
+//     return item;
+// }
+
+function getRandomNumber(choices) {
+    return Math.floor(Math.random() * choices);
```

Comment:
so you call ```getRandomNumber()``` on line 28 with the argument ```opponents```. ```opponents``` is an array: ```['boot', 'roach', 'nuke']```. You can't multiply a number by an array (well, technically you can, but it returns NaN which is JavaScript for not a number).

I think you wanna change it to ```choices.length```, which is the number of items in the array.